packages/ui5-proxy-middleware/test/base/utils.test.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    getCorporateProxyServer,
+    isHostExcludedFromProxy,
+    proxyRequestHandler,
+    proxyResponseHandler
+} from '../../src/base/utils';
+
+describe('Utils', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('proxyResponseHandler', () => {
```

Comment:
Can you add some descriptions to these tests? For example, what is `proxyResponseHandler` meant to be doing here? Given what input?

Reply:
OK, will do

Reply:
Done